tests/testthat/test-ensembleMethods.R

```diff
@@ -255,75 +250,74 @@ test_that("Do model results in caretEnsemble match component models - classifica
 
 test_that("Do model results in caretEnsemble match component models - regression", {
   skip_on_cran()
-  ens.reg <- caretEnsemble(models.reg, trControl=trainControl(method="none"))
-  models.class2 <- models.class[c(2:4)]
-  class(models.class2) <- "caretList"
-  ens.class <- caretEnsemble(models.class2, trControl=trainControl(method="none"))
+  ens.reg <- caretEnsemble(models.reg, trControl = trainControl(method = "none"))
+  ens.class <- caretEnsemble(models.class, trControl = trainControl(method = "none"))
   newDat <- ens.class$models[[3]]$trainingData
   newDat[2, 2] <- NA
   newDat[3, 3] <- NA
   newDat[4, 4] <- NA
   newDat <- newDat[1:10, ]
-  # expect_error(predict(ens.class, newdata = newDat, return_weights=TRUE, se=FALSE)) # BUG
```

Comment:
fixed this bug by re-adding the random forest model to the ens.class (note that the rest are ens.reg).  The "bug" was because the random forest model is the only one that can't handle these NAs and thus returns an error.

So in the old test, no RF meant no error.  Now that every test case has an RF, they all fail.